packages/babel-parser/src/parser/expression.js

```diff
@@ -1290,16 +1263,75 @@ export default class ExpressionParser extends LValParser {
             lookaheadCh === charCodes.greaterThan // Fragment <>
           ) {
             this.expectOnePlugin(["jsx", "flow", "typescript"]);
+            break;
+          } else {
+            throw this.unexpected();
           }
+        } else {
+          throw this.unexpected();
         }
```

Comment:
What are these changed needed for?

Reply:
￼For reasons that I couldn't figure out, the linter was complaining about missing `break`s after my changes *higher up*, so I put these lines in to quiet the linter. 

I can revisit this code and figure out if there’s a way to avoid the linter error without these changes.

Reply:
Specifically, it’s complaining: “error: Expected a 'break' statement before 'default' (no-fallthrough)”. I’m not sure why this is happening only now.

Reply:
I removed the two new `throw` statements, but I’m just not able to remove that `break` without the linter complaining.